src/50_Samples_%26_Templates/Product_Page.html

```diff
@@ -751,10 +767,22 @@ <h4 class="p2">Products that have been bought together</h4>
         </a>
       </template>
     </amp-list>
-    <!--
-    The button `tap` action sets the value of the `relatedProductsSrc` to an extended list of related products.
-    -->
-    <button [disabled]="listSrcButtonDisabled" class="ampstart-btn caps m1 mb3" id="listSrcButton" on="tap:AMP.setState({listSrcButtonDisabled: true, product: {relatedProductsSrc: '/json/more_related_products.json'}})">Show more</button>
+    <form method="GET"
```

Comment:
Can you document the ampstate and the amp form?

Reply:
done